Mta 4938 (#3922)
scaleway/docs-content: environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator-estimation.mdx

---
meta:
  title: Understanding your environmental footprint estimation

>   description: This page explains how to understand the environmental footprint estimation of your Elastic Metal.

Author: ```suggestion
  description: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
```

content:

>   h1: Understanding your environmental footprint estimation

Author: ```suggestion
  h1: Understanding environmental footprint estimation
```

>   paragraph: This page explains how to understand the environmental footprint estimation of your Elastic Metal.

Author: ```suggestion
  paragraph: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
```

tags: environmental-footprint impact environment datacentre
dates:
  validation: 2024-10-30
  posted: 2024-10-30
categories:
  - environmental-footprint
---

When you [order an Elastic Metal server](/bare-metal/elastic-metal/how-to/create-server/), you can see the estimated impact of your Elastic Metal server of choice before you order. In this page we explain how the estimation is measured.

<Message type="note">
  The estimation described on this page is based on the calculation methodology detailed in the [Environmental Footprint calculation breakdown](/environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator) documentation page.
</Message>

## Availability Zone footprint estimation

> When you select the Availability Zone of your Elastic Metal, you can see the indication of quantity of CO2e emitted per kW consumed per zone.

Reviewer: ```suggestion
When you select your Elastic Metal server's Availability Zone, you can see the quantity of CO2e emitted per kW consumed per zone.
```

<Lightbox src="scaleway-env-impact-estimation-az.webp" alt="screenshot of the Scaleway console showing the different availability zones in which you can create an Elastic Metal server. For each AZ, the quantity of CO2e emitted per kW consumed in shown in a green label" />

<Message type="tip">
  When creating an Elastic Metal server, you can click the **Low CO2 only** toggle to show only the regions that have low CO2 emission rates.
</Message>

> This data is determined by the [PUE (Power Usage Effectiveness)](/environmental-footprint/environmental-footprint/concepts/#pue-power-usage-effectiveness) of the datacenter and the [energy mix](#energy-mix) of the country in which the datacenter is located.

Reviewer: anchor does not exist

Author: oops

<Message type="note">
  We use [Ember's](https://ember-climate.org/) Electricity Data Explorer to determine the impact linked to the energy mix. You can also refer to [Electricity Maps' Environmental Impact by Zone map](https://app.electricitymaps.com/map) for a visual representation of the energy mix of each region, including France, Poland, and the Netherlands.
</Message>

## Elastic Metal offers footprint estimation

> For Elastic Metal offers, the environmental footprint estimation considers the impact of the **manufacturing** and **use** of all the elements that allow the service to work, such as network, cross-IT equipment and stock, and non-IT elements, Availability Zone  for example.

Reviewer: Is this correct to mention AZ in here?

---
meta:

>   title: Understanding your environmental footprint estimation

Author: ```suggestion
  title: Understanding environmental footprint estimation
```

<Message type="note">
  We use [Ember's](https://ember-climate.org/) Electricity Data Explorer to determine the impact linked to the energy mix. You can also refer to [Electricity Maps' Environmental Impact by Zone map](https://app.electricitymaps.com/map) for a visual representation of the energy mix of each region, including France, Poland, and the Netherlands.
</Message>

The estimation is represented by three green leaf icons:

  - When the three leafs are dark green, this means that the AZ has the lowest quantity of CO2e emitted per kW consumed
  - When two leafs are dark green, this means this that the AZ has a low quantity of CO2e emitted per kW consumed
  - When only one leaf is dark green, the AZ has bigger quantity of CO2e emitted per kW consumed

<Message type="note">
  When you hover your cursor over the leaves or when you select the AZ, you can see the detail of the environmental footprint estimation in kgCO₂e.
</Message>

## Elastic Metal offers footprint estimation

> For Elastic Metal offers, the environmental footprint estimation considers the impact of the **manufacturing** and **use** of all the elements that allow the service to work, such as network, cross-IT equipment and stock, and non-IT elements, for example.

Reviewer 2: ```suggestion
For Elastic Metal offers, the environmental footprint estimation considers the impact of the **manufacturing** and **use** of all the elements that allow the service to work, such as network, cross-IT equipment and stock, and non-IT elements, for example. The Az you chose in the previous step will impact the **use** part by multiplying each kw consumed by each of these elements by the value indicated.
```

We estimate the measure the server's electricity consumption and then calculate its estimated impact based on the energy mix of the selected zone and the datacenter's PUE.

The estimation is represented by three green leaf icons:

<Lightbox src="scaleway-select-a-server-env-foot.webp" alt="screenshot of the offer selection section of the Elastic Metal order page in the Scaleway console showing the different offers available to order. For each offer, three green leaves indicate the level of environmental impact - the system is explained below." />

  - When the three leafs are dark green, this means that the offer has the lowest environmental impact
  - When two leafs are dark green, this means this that the offer has a low environmental impact
  - When only one leaf is dark green, the offer has bigger environmental impact

<Message type="note">
  When you hover your cursor over the leaves or when you select an offer, you can see the detail of the environmental footprint estimation in kgCO₂e per hour or per month, depending on the billing method chosen.
</Message>

> A summary of the total environmental footprint estimation for the selected offer is shown before the order summary.

Reviewer 2: ```suggestion
A summary of the total environmental footprint estimation for the selected offer is shown before the order summary. We also add an indication to help you imagine what this impact is, to better understand.
```

<Lightbox src="scaleway-env-footprint-order-summary.webp" alt="screenshot of the Scaleway console showing an order summary of an Elastic Metal server, including a message box where the environmental footprint estimation is indicated in kgCO₂e" />

<Message type="important">

>   The AZ footprint estimation is is a straightforward calculation of CO₂ emissions per kW of electricity consumed, based on the energy mix and the data center’s Power Usage Effectiveness (PUE) specific to that AZ. For example, in an AZ like PAR-2, each kW consumed may have an impact of 0.065 kgCO₂e. <br /><br />

Reviewer 2: Should be at the begining  in "## Availability Zone footprint estimation"

---
meta:
  title: Understanding environmental footprint estimation
  description: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
content:
  h1: Understanding environmental footprint estimation
  paragraph: This page explains how to understand the environmental footprint estimation of Elastic Metal servers.
tags: environmental-footprint impact environment datacentre
dates:
  validation: 2024-10-30
  posted: 2024-10-30
categories:
  - environmental-footprint
---

> When you [order an Elastic Metal server](/bare-metal/elastic-metal/how-to/create-server/), you can see the estimated impact of your Elastic Metal server of choice before you order. In this page we explain how the estimation is measured.

Reviewer: ```suggestion
When you [order an Elastic Metal server](/bare-metal/elastic-metal/how-to/create-server/), you can see the estimated environmental impact of your Elastic Metal server of choice before you order. In this page we explain how the estimation is measured.
```

<Message type="note">
  The estimation described on this page is based on the calculation methodology detailed in the [Environmental Footprint calculation breakdown](/environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator) documentation page.
</Message>

## Availability Zone footprint estimation

> When you select your Elastic Metal server's Availability Zone, you can see the quantity of CO2e emitted per kW consumed for each zone.

Reviewer 3: Should we explain that CO2e = CO2 equivalent (the first time it appears on the page)?

Author: ```suggestion
When you select your Elastic Metal server's Availability Zone, you can see the quantity of CO2 equivalent (CO2e) emitted per kW consumed for each zone.
```

<Message type="note">
  We use [Ember's](https://ember-climate.org/) Electricity Data Explorer to determine the impact linked to the energy mix. The energy mix can be very different from one country to another depending on the origin of the electricity used. We can see on [Electricity Maps' Environmental Impact by Zone map](https://app.electricitymaps.com/map) the significant differences that there can be between countries.
</Message>

The estimation is represented by three green leaf icons:

  - When the three leaves are dark green, this means that the AZ has the lowest quantity of CO2e emitted per kW consumed, and each kw consumed in your Elastic Metal has a lower impact.
  - When two leaves are dark green, this means that the AZ has a low quantity of CO2e emitted per kW consumed
  - When only one leaf is dark green, the AZ has a bigger quantity of CO2e emitted per kW consumed

<Message type="note">

>   When you hover your cursor over the leaves or when you select the AZ, you can see the detail of the environmental footprint estimation in kgCO₂e.

Reviewer 2: ```suggestion
  When you hover your cursor over the leaves or when you select the AZ, you can see the detail of the environmental footprint estimation in kgCO₂e/kw.
```

</Message>

<Message type="important">
  The AZ footprint estimation is a straightforward calculation of CO₂ emissions per kW of electricity consumed, based on the energy mix and the data center’s Power Usage Effectiveness (PUE) specific to that AZ. For example, in an AZ like PAR-2, each kW consumed may have an impact of 0.065 kgCO₂e. <br /><br />

  Meanwhile, the offer footprint estimation is a more complex calculation that combines the AZ-specific data with additional factors relevant to the specific offer. This calculation considers not just energy consumption but also other elements, which are outlined in the [Environmental Footprint calculation breakdown](/environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator) documentation page, to reflect a comprehensive impact specific to the offer rather than just adding up kW-based impacts.
</Message>

## Elastic Metal offers footprint estimation

> For Elastic Metal offers, the environmental footprint estimation considers the impact of the **manufacturing** and **use** of all the elements that allow the service to work, such as network, cross-IT equipment and stock, and non-IT elements, and the manufacturing and use of the server itself, for example. The AZ you chose in the previous step will impact the **use** by multiplying each kw consumed by each of these elements by the value indicated.

Reviewer 2: ```suggestion
For Elastic Metal offers, the environmental footprint estimation considers the impact of the **manufacturing** and **use** of all the elements that allow the service to work, such as network, cross-IT equipment and stock, and non-IT elements in addition to the manufacturing and use of the server itself. The AZ you chose in the previous step will impact the **use** by multiplying each kw consumed by each of these elements by the value indicated.
```

<Message type="note">
  For a detailed report of how the environmental impact of Scaleway services is calculated, refer to the [Environmental Footprint calculator breakdown](/environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator) documentation page.
</Message>

> For usage impact, We base ourselves on an estimate of electricity consumption. This estimation is based on tests carried out by the Hardware team at Scaleway. Each range of offers was tested using different compute configurations to ensure coverage of a variety of possible power consumption setups.

Reviewer: ```suggestion
For usage impact, We base ourselves on an estimate of electricity consumption. This estimation is based on tests carried out by the hardware team at Scaleway. Each range of offers was tested using different compute configurations to ensure coverage of a variety of possible power consumption setups.
```

The estimation is represented by three green leaf icons:

<Lightbox src="scaleway-select-a-server-env-foot.webp" alt="screenshot of the offer selection section of the Elastic Metal order page in the Scaleway console showing the different offers available to order. For each offer, three green leaves indicate the level of environmental impact - the system is explained below." />

  - When the three leaves are dark green, this means that the offer has the lowest environmental impact
  - When two leaves are dark green, this means that the offer has a low environmental impact
  - When only one leaf is dark green, the offer has a bigger environmental impact

<Message type="note">
  When you hover your cursor over the leaves or when you select an offer, you can see the detail of the environmental footprint estimation in kgCO₂e per hour or per month, depending on the billing method chosen.
</Message>

A summary of the total environmental footprint estimation for the selected offer is shown before the order summary. We also add an indication to help you imagine what this impact is, such as `≈ 35 km by car`.

> <Lightbox src="scaleway-env-footprint-order-summary.webp" alt="screenshot of the Scaleway console showing an order summary of an Elastic Metal server, including a message box where the environmental footprint estimation is indicated in kgCO₂e" />

Reviewer 2: it's not a message box anymore

The estimation is represented by three green leaf icons:

<Lightbox src="scaleway-select-a-server-env-foot.webp" alt="screenshot of the offer selection section of the Elastic Metal order page in the Scaleway console showing the different offers available to order. For each offer, three green leaves indicate the level of environmental impact - the system is explained below." />

  - When the three leaves are dark green, this means that the offer has the lowest environmental impact
  - When two leaves are dark green, this means that the offer has a low environmental impact
  - When only one leaf is dark green, the offer has a bigger environmental impact

<Message type="note">
  When you hover your cursor over the leaves or when you select an offer, you can see the detail of the environmental footprint estimation in kgCO₂e per hour or per month, depending on the billing method chosen.
</Message>

A summary of the total environmental footprint estimation for the selected offer is shown before the order summary. We also add an indication to help you imagine what this impact is, such as `≈ 35 km by car`.

> <Lightbox src="scaleway-env-footprint-order-summary.webp" alt="screenshot of the Scaleway console showing an order summary of an Elastic Metal server, including a message box where the environmental footprint estimation is indicated in kgCO₂e" />

Author: ```suggestion
<Lightbox src="scaleway-env-footprint-order-summary.webp" alt="screenshot of the Scaleway console showing an order summary of an Elastic Metal server, including a dropdown where the environmental footprint estimation is indicated in kgCO₂e/month" />
```
